docs/_documentations/mdt-che-installinfo.md

```diff
@@ -139,12 +139,12 @@ Go to **View**>**Find Command…**>**Codewind: Build**.
 ## Updating the version
 Restart the Codewind workspace in Che. Che automatically pulls the newest version of Codewind and the Theia extension.
 
-# Installing Codewind on OpenShift
+## Installing Codewind on OpenShift
 
-## Codewind OpenShift Do (odo) extension 
-See the [`codewind-odo-extension` repository](https://github.com/eclipse/codewind-odo-extension) for the extension to Codewind that provides support for OpenShift projects.
+### Codewind OpenShift Do (odo) extension 
+For details about the Codewind extension that provides support for OpenShift projects, see the [`codewind-odo-extension` repository](https://github.com/eclipse/codewind-odo-extension). 
 
-## Adding rules to support the Codewind odo extension
+### Adding rules to support the Codewind odo extension
```

Comment:
We can move this under `After installing Codewind` heading for adding additional rules to support Codewind odo extension on post-installation of Codewind

Reply:
Moved to under `After installing Codewind`.